examples/tutorials/provision.ipynb

```diff
@@ -23,7 +23,12 @@
     "\n",
     "We like to put the provision files under\n",
     "\n",
-    "```/tmp/workspace```\n"
+    "```/tmp/workspace```\n",
+    "\n",
+    "\n",
+    ">**Warning**\n",
+    ">\n",
+    ">Provision and Start the NVFLARE must operate from a **terminal**, not within the notebook"
```

Comment:
if all these instructions are recommended to run in terminal but NOT notebook,
maybe we remove this notebook or rename this file to be a README or doc?